app-mobilephone/gammu/gammu-1.38.5-r1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="A tool to handle your cellular phone"
+HOMEPAGE="https://wammu.eu/gammu/"
+SRC_URI="https://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="bluetooth curl dbi debug irda mariadb mysql nls odbc postgres usb"
+
+COMMON_DEPEND="
+	dev-libs/glib:2=
+	virtual/libgudev:=
+	bluetooth? ( net-wireless/bluez:= )
+	curl? ( net-misc/curl:= )
+	dbi? ( >=dev-db/libdbi-0.8.3:= )
+	mariadb? ( dev-db/mariadb-connector-c:=[mysqlcompat] )
```

Comment:
I'm not going to merge this until https://bugs.gentoo.org/668560 is resolved.

Reply:
kill mariadb till 1.40 version